phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainerToolsTab.java

```diff
@@ -59,7 +59,7 @@ private void populate() {
         runExecutable.setOnMouseClicked(event -> {
             this.lockAll();
             FileChooser fileChooser = new FileChooser();
-            fileChooser.setTitle(tr("Choose executable"));
+            fileChooser.setTitle(tr("Choose executable…"));
```

Comment:
I'm not sure about this one. There seem to be different style guides (see https://en.wikipedia.org/wiki/Ellipsis). In the scripts repository, we use space and then 3 dots (`"Please wait ..."`). I think we should do the same here to be consistent.

Reply:
done

Reply:
You're still using the 3 dot symbol instead of 3 dots. Can we please use 3 dots to be consistent with scripts and then have a new issue for this topic like I said below?

Reply:
...

Reply:
@plata updated all like you want. Can this PR now be approved? When will Travis CI be working again? Thank you

Reply:
I've implemented the fix for Travis. Only waiting for approval by @madoar.